src/main/java/org/jabref/logic/xmp/XmpUtilWriter.java

```diff
@@ -159,6 +171,26 @@ private static void writeDublinCore(PDDocument document,
         catalog.setMetadata(metadataStream);
     }
 
+    public static String generateXmpString(List<BibEntry> entries) {
+        XMPMetadata meta = XMPMetadata.createXMPMetadata();
+        for (BibEntry entry : entries) {
+            DublinCoreSchema dcSchema = meta.createAndAddDublinCoreSchema();
+            XmpUtilWriter.writeToDCSchema(dcSchema, entry, null, Globals.prefs.getXMPPreferences());
```

Comment:
Globals should never be asked. The XMPPreferences should be passed to the method, not taken from globals!